docs/guidance/mobilenet-rayservice.md

```diff
@@ -0,0 +1,42 @@
+# RayService: MobileNet example
+
+Both Python scripts for Ray Serve and sending requests are in the repository [kevin85421/ray-serve-examples](https://github.com/kevin85421/ray-serve-examples).
```

Comment:
```suggestion
The Python files for the Ray Serve application and its client are in the repository [kevin85421/ray-serve-examples](https://github.com/kevin85421/ray-serve-examples).
```

Reply:
Can you change the repo to the `ray-project` repo that you put the files in, instead of your personal GitHub?

Reply:
Could you make this a `Note`? Something like [this](https://docs.ray.io/en/releases-2.5.1/serve/performance.html#performance-tuning) from the Ray docs:

<img width="682" alt="Screen Shot 2023-06-20 at 4 47 00 PM" src="https://github.com/ray-project/kuberay/assets/92341594/f5b12438-c4d8-4bfc-bd45-18bfb8c14f2a">

Reply:
> Can you change the repo to the ray-project repo that you put the files in, instead of your personal GitHub?

I apologize for this. I have already made the change but forgot to push it to the remote repository.

Reply:
No worries, thanks for making the change!

Reply:
Solved by https://github.com/ray-project/kuberay/pull/1175/commits/4bba967fcf2e7a145a4b404d8e91fa3ff6bc6641.